parm/config/gfs/config.base.emc.dyn

```diff
@@ -53,7 +53,7 @@ export NOSCRUB="@NOSCRUB@"
 export BASE_GIT="@BASE_GIT@"
 
 # Toggle to turn on/off GFS downstream processing.
-export DO_BUFRSND="NO"     # BUFR sounding products
+export DO_BUFRSND="YES"     # BUFR sounding products
```

Comment:
Did you actually want to change the default behavior to make bufr soundings? Or is this an accidental commit?

Reply:
Changed this temporarily for the CI tests so the postsnd job is included in the gfs suites. Will change back before merging the PR.

Reply:
After submitting, it dawned on me that was the likely reason. Just leave this as unresolved until you change back.

Reply:
I have reverted it back to "NO" at https://github.com/NOAA-EMC/global-workflow/pull/1834/commits/139a5499e1f61400bd00d1a5cb97225c692af180. Will now resolve this conversation and re-request your review.